releasenotes/notes/gate-cutting-workflow-ebbedbdb1313b258.yaml

```diff
@@ -21,21 +21,21 @@ features:
   - |
     Addition of :mod:`~circuit_knitting_toolbox.circuit_cutting.qpd` package which allows for easy
     transformation of elements of quantum circuits (gates and wires) into elements which
-    probabilistically decompose to a particular set of gates with respect to a given probabilistic basis. See 
+    decompose to a particular set of gates with respect to a given probabilistic basis. See 
     :class:`~circuit_knitting_toolbox.circuit_cutting.qpd.QPDBasis` and 
     :class:`~circuit_knitting_toolbox.circuit_cutting.qpd.BaseQPDGate` for more information.
-
-    Addition of :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_decomposition`, :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_decomposition`,
-    and :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_decomposition` modules. These modules
+  - |
+    Addition of :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_decomposition`, :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_execution`,
+    and :mod:`~circuit_knitting_toolbox.circuit_cutting.cutting_reconstruction` modules. These modules
     provide several functions which allow for easy implementation of a gate cutting workflow, namely,
-    the `partition_problem`, `decompose_gates`, `execute_experiments`, and `reconstruct_expectation_value` functions.
+    the :func:`partition_problem`, :func:`decompose_gates`, :func:`execute_experiments`, and :func:`reconstruct_expectation_value` functions.
+
 issues:
   - |
-    The :mod:`~circuit_knitting_toolbox.circuit_cutting` package only supports ``PauliList`` observables for calculating expectation values. Support for calculating
+    The :mod:`~circuit_knitting_toolbox.circuit_cutting` package only supports :class:`~qiskit.quantum_info.PauliList` observables for calculating expectation values. Support for calculating
     expectation values for more observable types, including :class:`~qiskit.quantum_info.SparsePauliOp`, is expected no sooner than 0.3.0.
   - |
-    The Qiskit and Qiskit Aer samplers do not support mid-circuit measurements in statevector mode. For more on reconstructing exact
-    expectation values in a circuit cutting workflow, check out our `how-to guide <https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/tree/main/docs/circuit_cutting/how-tos/how_to_generate_exact_quasi_dists_from_sampler.ipynb>`_.
+    The ``Sampler``\ s from Qiskit and Qiskit Aer do not support mid-circuit measurements in statevector mode. For more on generating exact quasi-distributions using the :class:`~qiskit.primitives.BaseSampler` interface, check out our `how-to guide <https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/tree/main/docs/circuit_cutting/how-tos/how_to_generate_exact_quasi_dists_from_sampler.ipynb>`_.
```

Comment:
If you try to use :mod: to describe the two primitives packages or :class: to describe the two Samplers you end up with a rendering of "The Samplers from *primitives* and primitives*" or "The *Sampler* and *Sampler* classes do not support...".

The clearest way to write it such that it renders clearly is to just double-backtick Sampler and don't get fancy with :mod: and :class:, since the Sampler and primitives namespaces are identical and it renders confusingly